content/en/community/_index.md

```diff
@@ -63,7 +63,9 @@ questions:
 You can also chat with other Etcd users and contributors in the #sig-etcd slack
```

Comment:
Nit: etcd should always be lowercase, let's fix this while we are updating the page.

```suggestion
You can also chat with other etcd users and contributors in the #sig-etcd slack
```